### PR TITLE
8161 sco tasks

### PR DESCRIPTION
**Pull request by kellykelly3**

### CODE REVIEW

File: Core/src/org/sleuthkit/autopsy/mainui/nodes/FileNode.java [hunk 1]
```diff
@@ -18,25 +18,34 @@
  */
 package org.sleuthkit.autopsy.mainui.nodes;
 
+import java.lang.ref.WeakReference;
+import java.util.ArrayList;
 import java.util.List;
 import java.util.Optional;
+import java.util.logging.Level;
+import java.util.logging.Logger;
```
Comment (dsmyda) [inline Core/src/org/sleuthkit/autopsy/mainui/nodes/FileNode.java:26]: Wrong logger :)

File: Core/src/org/sleuthkit/autopsy/mainui/sco/SCOFetcher.java [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+ */
+package org.sleuthkit.autopsy.mainui.sco;
+
+import java.lang.ref.WeakReference;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.concurrent.ExecutionException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
```
Comment (dsmyda) [inline Core/src/org/sleuthkit/autopsy/mainui/sco/SCOFetcher.java:27]: Wrong logger :)

File: Core/src/org/sleuthkit/autopsy/mainui/nodes/FileNode.java (hunk 1 above)
Comment (eugene7646) [reply Core/src/org/sleuthkit/autopsy/mainui/nodes/FileNode.java:26]: @dsmyda DANNY!!!! :) How have you been? Well it's nice of you to pay attention! ;) And yes, @kellykelly3 we should indeed use org.sleuthkit.autopsy.coreutils.Logger instead :)

Comment (dsmyda) [reply Core/src/org/sleuthkit/autopsy/mainui/nodes/FileNode.java]: Hey @eugene7646, I've been good! How are you doing?

And haha no problem, always happy to help! 

And I just want to say - big shoutout to the entire team, y'all are doing great stuff and handling lots of code :)

**Comment by kellykelly3** [comment]
This PR is covered by the follow on branch https://github.com/sleuthkit/autopsy/pull/7439

**Pull request closed without merging**